src/Tizen.Network.Stc/Tizen.Network.Stc/StcManager.cs

```diff
@@ -0,0 +1,120 @@
+        /// </summary>
+        protected override bool ReleaseHandle()
+        {
+            Interop.Stc.Deinitialize(this.handle);
+            this.SetHandle(IntPtr.Zero);
+            return true;
+        }
+    }
+
+    /// <summary>
+    /// A class which is used to manage Smart Traffic control (Stc).<br/>
+    /// </summary>
+    public static class StcManager
+    {
+        /// <summary>
+        /// Gets the Stc safe handle.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <returns>The instance of the SafeStcHandle.</returns>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown when the Stc is not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when the permission is denied.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal static SafeStcHandle GetStcHandle()
+        {
+            return StcManagerImpl.Instance.GetSafeHandle();
+        }
+
+        /// <summary>
+        /// Gets the statistics information of an application, which matched the rule, asynchronously. The rule must contain an Application ID.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <returns>The Statistics information of the application that used network in between timestamps specified.</returns>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown when the Stc is not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when the permission is denied.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        public static Task<NetworkStatistics> GetStatisticsAsync(StcRule rule)
+        {
+            return StcManagerImpl.Instance.GetStatisticsAsync(rule);
+        }
+
+        /// <summary>
+        /// Gets statistics information of each application that used network in between specified timestamps and matches the given rule, asynchronously.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <returns>A list of the NetworkStatistics objects.</returns>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown when the Stc is not supported.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when the permission is denied.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        public static Task<IEnumerable<NetworkStatistics>> GetAllStatisticsAsync(StcRule rule)
```

Comment:
Parameters of methods must have valid documentation. Method has 1 parameter, 0 are documented.

Reply:
Done.